test/test_serialization.h

```diff
@@ -769,6 +808,7 @@ TEST(serialization, sequential_to_json) {
   net1 << fully_connected_layer(10, 100) << tanh_layer()
        << dropout_layer(100, 0.3f, net_phase::test)
        << fully_connected_layer(100, 9) << softmax()
+       << recurrent_cell_layer(9, 9, false, new elu_layer)
```

Comment:
Since you have increased a layer in here, you should probablly update the teste code below.

I'd use a loop like this.
```cpp
for (size_t i=0; i<net1.layer_size(); ++i) {
  EXPECT_EQ(net1[i]->in_shape(), net2[i]->in_shape());
  EXPECT_EQ(net1[i]->layer_type(), net2[i]->layer_type());
}
```